src/SearchFilter.cc

```diff
@@ -49,7 +49,7 @@ void SearchFilter::guessFilterMode()
 
     if ( _filterMode == ExactMatch && _pattern.startsWith( "=" ) )
     {
-        _pattern.remove( QRegExp( "^=" ) );
+        _pattern.remove( 0, 1 );
```

Comment:
Qt 6.5 or later has `QString::removeFirst()` which is more speaking, but we can't use that yet. But a `// FIXME: Use _pattern.removeFirst()`  as a reminder would be good.